misk-hibernate/src/test/resources/moviestestmodule-common.yaml

```diff
@@ -1,8 +1,13 @@
-data_source:
+vitess_data_source:
   type: VITESS
   username: vt_app
   password: whatever
   vitess_schema_resource_root: classpath:/misk/hibernate/moviestestmodule-schema
+vitess_mysql_data_source:
+  type: VITESS_MYSQL
```

Comment:
our configs are auto-provisioned, how were you envisioning this to work? Generate another copy of each db for VITESS_MYSQL?

Reply:
Yeah it will be a gradual roll out. I imagine we add an option to the hcl which makes it generate `VITESS_MYSQL` config instead of `VITESS` and then we roll it one app the at the time and then eventually make it the default. It should be completely transparent to the app. You shouldn't even need to recompile (except to include this version of misk of course).

But first of all I want to make some scale/perf testing on this using Arnhem. It's still somewhat experimental at this stage I should say.